kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/OperationSupport.java

```diff
@@ -776,4 +771,28 @@ public <R1> R1 restCall(Class<R1> result, String... path) {
     }
   }
 
+  /**
+   * Send a raw request - where the type should be one of String, Reader, InputStream
+   */
+  public <R1> R1 handleRaw(Class<R1> result, String uri, String method, Object payload) {
+    try {
+      if (uri.startsWith("/")) {
+        // master ends with slash
+        uri = config.getMasterUrl() + uri.substring(1, uri.length());
+      }
+      String body = null;
+      if (payload instanceof String) {
+        body = (String) payload;
+      } else if (payload != null) {
+        body = Serialization.asJson(payload);
+      }
+      HttpRequest request = httpClient.newHttpRequestBuilder().uri(uri).method(method, JSON, body).build();
```

Comment:
in relation to my previous comment about security.

In which cases do we might want to perform a request to a non-relative URL?

Reply:
@shawkins, I'm going to merge as it is, but I think it's important to clarify this point. If there's no clear case, we might want to limit this to access only cluster-scoped URIs in order to prevent sending sensitive data to other locations.